lib/global_id/uri/gid.rb

```diff
@@ -154,6 +157,13 @@ def validate_component(component)
           "Expected a URI like gid://app/Person/1234: #{inspect}"
       end
 
+      def validate_model_id(model_id)
+        return model_id unless model_id.blank?
+
+        raise MissingModelId, "Unable to create a global id for #{model_name} " \
```

Comment:
model_name will be nil here. Inject model_name as a second parameter and upgrade the post fix if on line 147 to a standard if with the validations on separate lines.